pkg/operation/shoot/shoot.go

```diff
@@ -195,6 +196,9 @@ func (b *Builder) Build(ctx context.Context, c client.Client) (*Shoot, error) {
 	}
 
 	shoot.KonnectivityTunnelEnabled = gardenletfeatures.FeatureGate.Enabled(features.KonnectivityTunnel) && kubernetesVersionGeq118
+	if konnectivityTunnelEnabled, err := strconv.ParseBool(shoot.Info.Annotations["alpha.featuregates.shoot.gardener.cloud/konnectivity-tunnel"]); err == nil && kubernetesVersionGeq118 {
```

Comment:
Can you add `alpha.featuregates.shoot.gardener.cloud/konnectivity-tunnel` [here](https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/constants/types_constants.go)?